playbooks/roles/mysql/files/edxapp.sql

```diff
@@ -117,7 +117,7 @@ SELECT
     SELECT * FROM auth_user WHERE username='oxamaster'
   ) LIMIT 1
 );
-
+commit;
```

Comment:
nit: you may want to take an opportunity to improve the formatting a bit for this query file.